File: generator/sources/csharp/KalturaClient/KalturaServiceBase.cs
using System;
using System.Text;

namespace Kaltura
{
    public class KalturaServiceBase
    {
        #region Private Fields

        protected KalturaClient _Client;

        #endregion

        #region CTor

        public KalturaServiceBase(KalturaClient client)
        {
            this._Client = client;
        }

        #endregion
    }
}

File: generator/sources/java/src/com/kaltura/client/KalturaServiceBase.java
package com.kaltura.client;

/**
 * Common ancestor for all generated classes in the com.kaltura.client.services package.
 * 
 * @author jpotts
 *
 */
public class KalturaServiceBase {

	protected KalturaClient kalturaClient;
	
	public KalturaServiceBase() {		
	}
	
	public KalturaServiceBase(KalturaClient kalturaClient) {
		this.kalturaClient = kalturaClient;
	}
	
	public void setKalturaClient(KalturaClient kalturaClient) {
		this.kalturaClient = kalturaClient;
	}
	
}
